doc/WEBUI.md

```diff
@@ -202,6 +202,7 @@ single or double quotes.
   - `netdev`, `networkdevice` look for network devices (firewalls,
     routers, ...).
   - `phonedev` look for telephony devices.
+  - `cpe:[value] | cpe:[field=value](, ...)` look for a given cpe, by value or by field (type, vendor, component or product).
```

Comment:
Following our discussion, I updated the syntax to be `cpe(:type(:vendor(:product(:version)))` (you can see help.js for more information).